homeassistant/components/balboa/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "balboa",
+  "name": "Balboa Spa Client",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/balboa",
+  "requirements": [
+    "pybalboa==0.13"
+  ],
+  "codeowners": [
+    "@garbled1"
+  ],
+  "iot_class": "local_push"
+}
```

Comment:
You could make it even easier to discover a Balboa spa by adding:
`"dhcp": [
    {
      "macaddress": "001527*"
    }
  ]` to the manifest and then adding a dhcp discovery step to `config_flow.py`

Reply:
My plan was to add 30303 style discovery in a future PR.  For now I just want to get the same functionality that is in HACS ported over, and then improve from there.

Reply:
I'm glad it's finally getting brought into core. ^_^